server/admin_management/api/app/routers/router_access_control_privilege.py

```diff
@@ -45,6 +48,7 @@ def create_access_control_privilege_many(
     request: Request,
     token_claims: dict = Depends(jwt_validation.authorize),
     requester: Requester = Depends(get_current_requester),
+    target_user: TargetUser = Depends(target_user_bceid_search),
```

Comment:
"Depends" to make target_user available for "create_access_control_privilege_many" endpoint. The BCeID target_user has the business_guid searched.